README.md

```diff
@@ -231,3 +229,71 @@ on Red Hat Enterprise Linux 6
 
 The general usage is demonstrated in
 [selinux-playbook.yml](examples/selinux-playbook.yml) playbook.
+
+## rpm-ostree
```

Comment:
We can put this info into a separate .md file the same way we do with [e.g.](https://github.com/linux-system-roles/.github/blob/main/playbooks/templates/README-ansible.md).  It'd would be nice to keep such a specific info not in the main README, WDYT?

Reply:
That would make it much easier to keep it up-to-date, and not "pollute" the file with information not needed by users who aren't interested in ostree.  That brings up another point - we'll need to document this somehow in the collection https://galaxy.ansible.com/ui/repo/published/fedora/linux_system_roles/docs/ - we'll either need to add this to the collection readme, or have a separate file there too e.g. README-ostree

@briansmith0 wdyt?

Reply:
If you look at rhel-system-roles AH page, the main README.md is printed on the main page of the collection. Then, there is a **Documentation** tab where you can find all other md files, now they are roles' READMEs and CHANGELOGs. For visibility, it's better to put info about ostree is to put it to the main collection README.

Reply:
@spetrosi @briansmith0 please review the latest README.md and the new README-ostree.md

Reply:
> @spetrosi @briansmith0 please review the latest README.md and the new README-ostree.md

Those files look good to me.  Thanks!

Reply:
done.  And I will be adding this to https://github.com/linux-system-roles/.github along with some other changes for ostree